src/Strings.ts

```diff
@@ -263,8 +278,8 @@ export const strings = (locale: string) => {
         locale = 'zh-hant';
     else if (locale.startsWith('zh'))
         locale = 'zh-hans';
-    else if (locale.startsWith('cs'))
-        locale = 'cs-cz';
+	else if (locale.startsWith('cs'))
```

Comment:
undo indentation

Reply:
oic - you are using tabs, but the existing code uses spaces. Can you convert your tabs to spaces? Also in the strings.ts file.

Reply:
Sorry about that Dan.  I should have noticed the use of spaces vs tabs.  Is there a style doc somewhere?

Reply:
There isn't a style guide - just look around :-)